core/src/main/scala/org/broadinstitute/dsde/rawls/monitor/BillingAccountChangeSynchronizer.scala

```diff
@@ -204,7 +205,7 @@ final case class BillingAccountChangeSynchronizer(dataSource: SlickDataSource,
       v2Workspaces <- inTransaction {
         workspaceQuery
           .withBillingProject(billingProject.projectName)
-          .withoutGoogleProjectId(billingProject.googleProjectId)
+          .withVersion(WorkspaceVersions.V2)
```

Comment:
Ah my bad, this was a silly mistake. Hope it wasn't too difficult to spot.

Reply:
Awww, so nice of you to drop a note! We were trying not to bug you.

So there was no reason why you didn't check the version number? We were wondering if there was some transitory state where the version number might not be correct.

Reply:
Because I'm dumb

Reply:
And I didn't realise a workspace had a version number

Reply:
Ah, well if you didn't know it had a version number I understand the comparison to the billing project name! Unfortunately though this caused an issue for migrated workspaces that kept their original billing project name.